cinema/views.py

```diff
@@ -45,7 +45,7 @@ def put(self, request, pk):
 
     def patch(self, request, pk):
         serializer = GenreSerializer(data=request.data)
-        if serializer.is_valid(raise_exception=True):
+        if serializer.is_valid():
```

Comment:
use raise_exception=True and delete `if` and `return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)`